cfme/tests/infrastructure/test_host_devices.py

```diff
@@ -0,0 +1,32 @@
+
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon, toolbar
+from cfme.infrastructure.host import Host
+from cfme.web_ui import InfoBlock
+from cfme.fixtures import pytest_selenium as sel
+from cfme.configure.tasks import is_host_analysis_finished
+from utils.wait import wait_for
+
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                              ['openstack-infra'], scope='module')
+
+
+def get_integer_value(x):
```

Comment:
No, host does not have SummaryMixin ... yet. Once that one is in place, we can change this.